core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.spec.ts

```diff
@@ -156,6 +160,12 @@ describe('Conversation skin component', () => {
   let stateObjectFactory: StateObjectFactory;
   let platformFeatureService: PlatformFeatureService;
   let learnerDashboardBackendApiService: LearnerDashboardBackendApiService;
+  let conceptCardManagerService: ConceptCardManagerService;
+  let audioTranslationLanguageService: AudioTranslationLanguageService;
+  let translateService: TranslateService;
+
+  const WAIT_BEFORE_RESPONSE_FOR_STUCK_LEARNER_MSEC: number = 11000;
+  const WAIT_BEFORE_REALLY_STUCK_MSEC: number = 12000;
```

Comment:
These are already defined in `exploration-player-page.constants.ts`.

Reply:
Fixed.